tests/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ add_executable(kfs_fuse
         kfs_ftruncate_test.c
         kfs_readlink_test.c
         kfs_fsync_test.c
-        kfs_opendir_test.c)
+        kfs_opendir_test.c
```

Comment:
again, don't mix code from different PR

Reply:
i don't mix code, just removed ")" from line 35 and add it to line 36

Reply:
anyway this change releated to other stuff

Reply:
roger that, i will fix it in next PR